cypress/integration/RichTextEditor.spec.ts

```diff
@@ -269,6 +273,28 @@ describe('Rich Text Editor', () => {
 
         expectRichTextFieldValue(expectedValue);
       });
+
+      it('should unwrap blockquote', () => {
+        editor().click().typeInSlate('some text');
+
+        getQuoteToolbarButton().click();
+
+        const expectedQuoteValue = doc(
+          block(BLOCKS.QUOTE, {}, block(BLOCKS.PARAGRAPH, {}, text('some text', []))),
+          block(BLOCKS.PARAGRAPH, {}, text('', []))
+        );
+        expectRichTextFieldValue(expectedQuoteValue);
+
+        getHrToolbarButton().click();
+
+        const expectedHeadingValue = doc(
+          block(BLOCKS.PARAGRAPH, {}, text('some text', [])),
+          block(BLOCKS.HR, {}),
+          block(BLOCKS.PARAGRAPH, {}, text('', [])),
+          block(BLOCKS.PARAGRAPH, {}, text('', []))
```

Comment:
Who do we have two empty paragraphs now? Seems odd but I guess out of scope here.

Reply:
1 empty paragraph when adding blockquote and another one when adding HR. We could be more surgical and manage that but that's also how the current editor looks like when doing the same action.

Reply:
Yes, we should definitely improve where it makes sense and where it's no effort. The old editor definitely wasn't perfect.

At least it seems this is just a behavior when a blockquote gets unwrapped, we're not spamming the editor with unnecessary paragraphs when just inserting a hr so I don't think it's a common thing to happen :sweat_smile:

Reply:
Not sure what happened to my last comment but I said that the old editor behavior is not necessarily perfect but we are at least not inserting "new things", things will stay as is but again, it doesn't mean it's the right behavior. 

Inserting an empty paragraph for blockquote and HR below them is "necessary" otherwise the customer will not be able to move through rows. The problem is that we are not removing those empty paragraphs when unwrapping things, which is where we can do better.